src/main/webapp/js/instructorStudentRecords.js

```diff
@@ -22,10 +22,22 @@ $(document).ready(function() {
 
     $('.panel-heading.student_feedback').click(toggleSingleCollapse);
 
-    $('input[type=checkbox]').click(function() {
+    $('input[type=checkbox]').click(function(e) {
```

Comment:
Could you also fix the 2 lines below this? Or else there will be conflicts with @JosephineKwa 's PR for sure